lib/views/pages/home/content/map/map_screen.dart

```diff
@@ -29,14 +36,15 @@ class MapScreen extends ConsumerWidget {
               MapSelectionOptionChips(mapInfo: value),
               const Divider(key: dividerKey),
               //TODO: change the map when clicking on a selection option
```

Comment:
```suggestion
```
Not your responsibility, but isn't this TODO completed?

Reply:
yes I think so, a42759b3ec99b35c95f046d5d04c2e4fda332af7, (I pushed to the other map branch by accident, but we can leave it like this)

Reply:
Thanks